### PR TITLE
Added info to Showcase PR commit message

### PR DESCRIPTION
**Pull request by klinegareth**

### CODE REVIEW

**Comment by klinegareth** [comment]
Opening this PR to work on issue #334

**Comment by shiffman** [comment]
@klinegareth this is looking great! Let's add a label also? I'll create one.

File: netlify/functions/submission-background.js [hunk 1]
```diff
@@ -110,8 +110,13 @@ exports.handler = async function (event) {
...
   **/
   const prRes = await octokit.request(`POST /repos/${owner}/${repo}/pulls`, {
-    title: `Passenger showcase contribution from ${postInfo.authorName}`,
-    body: 'Yay!',
+    title: `Passenger Showcase contribution from ${postInfo.authorName}`,
+    body: `Thank you ${postInfo.authorName} for your contribution! A member of the Coding Train team will review it shortly.
+
+	<img width "400" alt="Showcase Screenshot" src=${postInfo.image}>
+
+	* [${postinfo.title}](${postInfo.url})
```
Comment (shiffman) [inline netlify/functions/submission-background.js:118]:
```suggestion
	* [${postInfo.title}](${postInfo.url})
```

**Comment by shiffman** [comment]
Hi @runemadsen! @klinegareth and I are a little stumped with this one, when I test the function in the deploy preview for 347 I get a "success" message on the page, but not pull request comes through. At the moment netlify is reporting the following message in the logs.

> Aug 5, 02:53:17 PM: [ERROR] Function logs are currently unavailable. We are working on resolving the issue.

I'll try again later, but when you are back online we might need help reviewing these changes before we merge.

**Comment by runemadsen** [comment]
This looks great! It sounds like maybe Netlify had problems, but let me know if we can help in any way.

**Comment by shiffman** [comment]
Ok, I went ahead and tested and got an error message! This is the relevant part I'm assuming:

```
"message\":\"body is too long (maximum is 65536 characters)\"}"," 
```

@klinegareth what do you think we added here that caused it to go above the limit?

This is the full error (with all base64 image characters removed) in case that's helpful.

```
Aug 15, 10:48:59 AM: 6b8b85e2 INFO   Handler called with:  {"challenge":"172-directional-boring","title":"Testing","url":"https://thecodingtrain.com/","image":"==>\\n\\n\\t* [Testing](https://thecodingtrain.com/)\\n\\t* [Daniel Shiffman](https://thecodingtrain.com/)\",\"head\":\"showcase-daniel-shiffman-1660574939\",\"base\":\"main\"}","request":{}},"stack":["HttpError: Validation Failed: {\"resource\":\"Issue\",\"code\":\"custom\",\"field\":\"body\",\"message\":\"body is too long (maximum is 65536 characters)\"}","    at /var/task/node_modules/@octokit/request/dist-node/index.js:86:21","    at processTicksAndRejections (internal/process/task_queues.js:95:5)","    at async Job.doExecute (/var/task/node_modules/bottleneck/light.js:405:18)"]}
Aug 15, 10:49:03 AM: 6b8b85e2 Duration: 3512.26 ms	Memory Usage: 80 MB	Init Duration: 306.18 ms	
```

**Comment by runemadsen** [comment]
Is it perhaps the image that is too large? This is really odd.

**Comment by shiffman** [comment]
Ahhh, ok I think I know what it is. It's the fact that we are trying to embed the image (using its base64 encoding) in the pull request comment itself. Let's remove that and see if it fixes it. We will need to research if the GitHub API will allow for media uploads which would then just be referenced via the URL. 

@klinegareth can you take that feature out and fix the conflicts?Then I think we will be all set. Let's then file an issue to revisit embedding the image later.

File: netlify/functions/submission-background.js [hunk 2]
```diff
@@ -110,8 +112,13 @@ exports.handler = async function (event) {
     Make a PR to main
   **/
   const prRes = await octokit.request(`POST /repos/${owner}/${repo}/pulls`, {
-    title: `Passenger showcase contribution from ${postInfo.authorName}`,
-    body: 'Yay!',
+    title: `Passenger Showcase contribution from ${postInfo.authorName}`,
+    body: `Thank you ${postInfo.authorName} for your contribution! A member of the Coding Train team will review it shortly.
+
+	<img width "400" alt="Showcase Screenshot" src=${postInfo.image}>
```
Comment (shiffman) [inline netlify/functions/submission-background.js:118]:
```suggestion
```

I believe it should work removing this image embed.

**Comment by shiffman** [comment]
Hi @klinegareth this is working now, but for some reason the text in the comment is showing in plain text rather than Markdown. I don't see an obvious reason why, can you check? See https://github.com/CodingTrain/thecodingtrain.com/pull/466.

Screen Shot 2022-08-19 at 5 46 24 PM

Also, let's change the subject to: `Showcase Submission for [VIDEO NAME HERE]`, we've got the author's name (and personal link) in the body of the message.

**Comment by dipamsen** [comment]
Because of the tab indentation before the bullet points, GitHub treats it as a code block.
image

**Comment by shiffman** [comment]
Looking good! @klinegareth do you know if it's easy to add the `showcase` label via the API?

**Comment by shiffman** [comment]
I'm going to go ahead and merge this and note some remaining elements that would be "nice to have" in a new issue. Thank you @klinegareth for this!!

**Pull request merged**